assignments/callbacks.js

```diff
@@ -19,34 +20,79 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
     console.log(first)
   });
 
-*/
+// */
 
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+   return cb(arr.length);
 }
+ getLength(items, function(len) {
+   console.log(len);
+ });
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length-1]);
 }
 
+last(items, function(end) {
+  console.log(end);
+});
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y);
 }
 
+sumNums(2, 3, function(sum) {
+  console.log(sum);
+});
+
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y);
 }
+multiplyNums(2, 3, function(times) {
+  console.log(times);
+});
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  let check = false;
+
+  for (let i = 0; i < list.length; i++) {
+    if (list[i] === item) {
+      check = true;
+    }
+  }
+
+  return cb(check);
 }
 
+contains('Gum', items, function(result){
+  console.log(result);
+});
+
 /* STRETCH PROBLEM */
 
 function removeDuplicates(array, cb) {
   // removeDuplicates removes all duplicate values from the given array.
   // Pass the duplicate free array to the callback function.
   // Do not mutate the original array.
+
+  let noDupes = [];
+  
+  noDupes = array.filter(function(item, index) {
+      return index <= array.indexOf(item); 
```

Comment:
So in case you didn't already know the reason this works is because indexOf returns -1 if the argument isn't in the array upon which the method is invoked.  .includes() is probably the best tool for this, or Set.

Great job though.